pkg/apis/infinispan/v1/types_util.go

```diff
@@ -0,0 +1,42 @@
+package v1
+
+// GetCondition return true if condition == status
```

Comment:
The comment does not match the method API.

Reply:
yep thanks